terraform/getproviders/lock.go

```diff
@@ -117,6 +119,33 @@ func updateProviderBlock(ctx context.Context, providerBlock *hclwrite.Block, pro
 	return nil
 }
 
+func getExistingHashes(providerBlock *hclwrite.Block, provider Provider) ([]Hash, error) {
+	var hashes []Hash
+
+	versionAttr := providerBlock.Body().GetAttribute("version")
+	if versionAttr != nil {
```

Comment:
To reduce nested `{` blocks, make sense to return `return hashes, nil` if versionAttr == nil?

Reply:
Indeed. Thanks for the tip.